### ISSUE TITLE
NTR: Tetrahydrobiopterin low in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8477]
New term request Tetrahydrobiopterin low in CSF (segal_150721222209): BH4; cerebrospinal fluid; sapropterin  UMLS: 2751159

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000764
- label: Decreased CSF tetrahydrobiopterin level
- parents: [Abnormal CSF biopterin concentration (HP:0040207)](https://hpo.jax.org/app/browse/term/HP:0040207)
- pmid: 
- def: Concentration of tetrahydrobiopterin in the cerebrospinal fluid (CSF) below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8477
- synonyms:
